packages/plugins/content/slate/src/hooks.js

```diff
@@ -76,23 +89,29 @@ export const html = new Html({
   parseHtml: parse5.parseFragment
 })
 
-const options = { terse: true }
+const options = {}
 
 export const createInitialState = () => ({
-  editorState: Raw.deserialize(
+  editorState: Value.fromJSON(
     {
-      nodes: [
-        {
-          kind: 'block',
-          type: P,
-          nodes: [
-            {
-              kind: 'text',
-              text: ''
-            }
-          ]
-        }
-      ]
+      document: {
+        nodes: [
+          {
+            object: "block",
+            type: "paragraph",
```

Comment:
Is it intentional that this is `"paragraph"` and not `P`?

Reply:
nope that's an error on my side, fixed